### ISSUE TITLE
mAP计算

### ISSUE THREAD

**Issue by 2018loveyyq** [csuhan/s2anet#27]
你好，我想问一下关于mAP的计算，在训练阶段的时候，看代码里显示的是DOTA数据集的train和val set都用了，请问在DOTA_devkit中的dota_evaluation_task1.py中的标注文件是什么，也是用val 数据集进行验证的吗？已经参与训练的数据再测试会不会使得结果不准确？
还有想知道作者这里的test的标注文件是什么？DOTA数据集test是没有标注的啊？
![image](https://user-images.githubusercontent.com/38448650/97127712-59caaf80-1775-11eb-82fa-1b7b07028742.png)

**Comment by csuhan** [comment]
Both training and validation sets are used for training, and the testing set is used for testing. See our paper for details.

**Comment by 2018loveyyq** [comment]
yes, I have seen it,but I want to know if you submit the evaluate results on test set to the evaluation server to get the&nbsp;final&nbsp;mAP&nbsp;in this&nbsp;paper?And what the&nbsp;follwing code for?





------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "csuhan/s2anet"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2020年10月26日(星期一) 上午10:39
收件人:&nbsp;"csuhan/s2anet"<s2anet@noreply.github.com&gt;;
抄送:&nbsp;"十二"<2539372545@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [csuhan/s2anet] mAP计算 (#27)





 
Both training and validation sets are used for training, and the testing set is used for testing. See our paper for details.

**Comment by csuhan** [comment]
1. We have stated in the footnote on page 8 of the paper. See [DOTA Results](https://captain-whu.github.io/DOTA/results.html) on Task1, Page 2 with the teamname `hanjiaming`.
2. All the results on DOTA, except Sec IV D, are trained on the training and validation sets, tested on the testing set.
3. DOTA is annotated by [our group](http://www.captain-whu.com), so we test our results locally, but submit the finnal results to the [online evaluation server](https://captain-whu.github.io/DOTA/evaluation.html).
4. All the codes used for data processing, training and testing are provided,  as well as pretrained models, one can easily verify the performance. Refer to [GETTING_STARTED.md](https://github.com/csuhan/s2anet/blob/master/docs/GETTING_STARTED.md).

**Comment by 2018loveyyq** [comment]
OK,thanks for your reply!




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "csuhan/s2anet"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2020年10月26日(星期一) 中午11:03
收件人:&nbsp;"csuhan/s2anet"<s2anet@noreply.github.com&gt;;
抄送:&nbsp;"十二"<2539372545@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [csuhan/s2anet] mAP计算 (#27)





  
We have stated in the footnote on page 8 of the paper. See DOTA Results on Task1, Page 2 with the teamname hanjiaming.
 
All the results on DOTA, except Sec IV D, are trained on the training and validation sets, tested on the testing set.
 
DOTA is annotated by our group, so we test our results locally, but submit the finnal results to the online evaluation server.
 
All the codes used for data processing, training and testing are provided,  as well as pretrained models, one can easily verify the performance. Refer to GETTING_STARTED.md.
